lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -94,13 +98,52 @@ def add_amount(post, amount)
         }
       end
 
-      def add_credit_card(post, creditcard)
+      def add_payment(post, payment, options)
+        if payment.is_a?(NetworkTokenizationCreditCard)
+          add_apple_google(post, payment, options)
+        else
+          add_credit_card(post, payment)
+        end
+      end
+
+      def add_apple_google(post, payment, options)
+        post[:processingInformation] = {
+          paymentSolution: PAYMENT_SOLUTION[payment.source]
+        }
+        # supported cards: JCB, VISA, MASTERCARD
+        post[:processingInformation][:commerceIndicator] = 'internet' unless card_brand(payment) == 'jcb'
+
+        # eFrocreIantdriacnastoacr tion that does not use 3-DSecure, set this field to internet.
+        # For a transaction that uses 3-D Secure, set this field to spa.
+        # post[:processingInformation][:commerceIndicator] = 'spa' if card_brand(payment) == 'master' # mc
```

Comment:
Do you have any reason for leaving these comments?

Reply:
I want to clarify the uses with 3ds and Mastercard, for apple/google pay transactions because of that the value of 'internet' must be replaced by 'spa'

Reply:
After a review, it looks like unnecessary the 'spa' flag is.